jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+	clearMocks: true,
+	restoreMocks: true,
+	resetModules: true,
+	testEnvironment: "node",
+	testPathIgnorePatterns: ["<rootDir>/node_modules", "<rootDir>/monorepo-lambda"],
```

Comment:
We don't need to test the `typescript-lambda` as well.

```suggestion
	testPathIgnorePatterns: ["<rootDir>/node_modules", "<rootDir>/monorepo-lambda", "<rootDir>/typescript-lambda"],
```